src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -43,10 +47,35 @@ interface ImplicitParam<T, K extends keyof T> {
   value: T[K]
 }
 
+interface ComputedParam<T, K extends keyof T> {
+  name: string
+  computeValue: (otherFields: Partial<T>) => T[K]
+}
+
 type AccountNumberSchema = {
   [Property in keyof FiatAccountSchemas[FiatAccountSchema.AccountNumber]]:
     | FormFieldParam
     | ImplicitParam<FiatAccountSchemas[FiatAccountSchema.AccountNumber], Property>
+    | ComputedParam<FiatAccountSchemas[FiatAccountSchema.AccountNumber], Property>
+}
+
+export function getAccountName({
+  accountNumber,
+  institutionName,
+}: {
+  accountNumber?: string
+  institutionName?: string
+}) {
+  if (institutionName && accountNumber) {
+    return institutionName + ' ' + getObfuscatedAccountNumber(accountNumber)
```

Comment:
The designs in figma this string always have parentheses as well 
```suggestion
    return `${institutionName} (${getObfuscatedAccountNumber(accountNumber)})`
```